package.json

```diff
@@ -8,16 +8,18 @@
   },
   "devDependencies": {
     "casperjs": "~1.1.0-beta3",
+    "jscs": "~2.6.0",
+    "jshint": "~2.9.1",
```

Comment:
When I do `make media/main-built.js` on this branch, I get an npm error saying "No compatible version found: jshint@~2.9.1"

It looks like the latest version of jshint is 2.9.1-rc1 right now: https://www.npmjs.com/package/jshint

Maybe they released 2.9.1 then retracted it? I think they've done something like that before.. https://twitter.com/JSHint/status/639564453852356608